### ISSUE TITLE
Medial epicondyle apophysitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10827]
**Preferred term label:**
Medial epicondyle apophysitis

**Synonyms**

**Definition (free text, please give PubMed ID)**
MRI showing increased fluid signal at the medial epicondyle with widening of the physis; ultrasound showing increased echogenicity at the medial epicondyle region

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Little league elbow 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I wonder if MEA is better characterized as a disease entity that has the phenotypes medial epicondyle point tenderness, medial elbow pain with valgus stress, and sometimes subtle elbow contractures.
https://www.ncbi.nlm.nih.gov/books/NBK570592/
The imaging findings are complex and I understand the suggestion to summaizre them, but essentially if we use this HPO term then we would be making the diagnosis already so it does not seem to fit?

**Comment by MickeySegal** [comment]
This issue is present for many findings that a radiologist can observe, e.g., https://hpo.jax.org/browse/term/HP:0003978

A reasonable approach is that if a radiologist can call a finding from the images alone it should be considered a finding and get an HPO term.

**Comment by pnrobinson** [comment]
@MickeySegal agreed.

Adding new term: 
- ID: HP:6001263
- label: Medial epicondyle apophysitis
- parents: [Abnormal joint morphology (HP:0001367)](https://hpo.jax.org/app/browse/term/HP:0001367)
- pmid: PMID:34033354
- def: Medial epicondyle apophysitis (MEA), also known as little league elbow, is an overuse injury prevalent among adolescent athletes whose sport involves repetitive overhand throwing, racket use, or other overhead arm motions. The medial epicondyle is a bony protrusion on the medial elbow with its own ossification center, separate from the main distal humeral physis, known as an apophysis. Medial epicondyle apophysitis occurs due to excessive and repetitive valgus stress placed on the apophysis before its closure. Over time, the repetitive strain results in a traction-type injury and may cause pathologic widening and inflammation of the apophysis or even an avulsion fracture.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10827
- synonyms: Little league elbow
